src/algokit_utils/application_client.py

```diff
@@ -1328,18 +1324,27 @@ def execute_atc(self, atc: AtomicTransactionComposer) -> AtomicTransactionRespon
     def _set_app_id_from_tx_id(self, tx_id: str) -> None:
         self.app_id = get_app_id_from_tx_id(self.algod_client, tx_id)
 
+    def resolve_signer_sender(
+        self, signer: TransactionSigner | None = None, sender: str | None = None
+    ) -> tuple[TransactionSigner, str]:
+        """Return signer and sender, using default values on client if not specified
+
+        Will use provided values if given, otherwise will fall back to values defined on client.
+        If no sender is specified then will attempt to obtain sender from signer
+
+        :raises Exception: If a signer or sender is not specified"""
+        return self._resolve_signer_sender(signer, sender)
+
+    # TODO: remove private implementation, kept in the 1.0.2 release to not impact existing beaker 1.0 installs
     def _resolve_signer_sender(
```

Comment:
you can add an alias to a bound method inside a class body fyi

Reply:
Done